docs/api_reference/src/quantities.tex

```diff
@@ -146,121 +2593,397 @@
+  {
+  }
+};
 
 }
 \end{codeblock}
 
+\pnum
 Let \tcode{\placeholder{Q}} be a specialization of \tcode{quantity}.
 \begin{itemize}
 \item
-If \tcode{Rep} is a scalar,
+If \tcode{Rep} is a scalar\iref{qty.set.traits},
 \tcode{\placeholder{Q}} represents a scalar quantity\irefiev{102-02-19}.
+% FIXME What if `Rep` is a complex number?
 \item
-If \tcode{Rep} is a vector,
+If \tcode{Rep} is a vector\iref{qty.set.traits},
 \tcode{\placeholder{Q}} represents a vector\irefiev{102-03-04}.
 % FIXME What if `Rep` is a tensor?
 \end{itemize}
 
-\rSec2[qty.point.type]{Class template \tcode{quantity_point}}
+\rSec2[qty.pt.concepts]{Quantity point concepts}
+
+\begin{itemdecl}
+template<typename T>
+concept @\deflibconcept{PointOrigin}@ = @\unspecnc@;
+\end{itemdecl}
+
+\rSec2[qty.pt]{Class template \tcode{quantity_point}}
 
 \begin{codeblock}
 namespace mp_units {
 
-export template<@\unspec@>
-class quantity_point { @\unspec@ };
+template<@\unspec@>
+class @\libglobal{quantity_point}@ { @\unspec@ };
 
 }
 \end{codeblock}
 
+\pnum
+\label{term.quantity.point.type}
 A \defnadj{quantity point}{type} is a specialization of \tcode{quantity_point}.
 Let \tcode{\placeholder{Q}} be a quantity point type.
 \tcode{\placeholdernc{Q}::point_origin} represents
-the origin point of a position vector\irefiev{102-03-15}.
+the origin point of a position vector\irefiev{102-03-15}
+or of a component\irefiev{102-03-10} thereof.
 \begin{itemize}
 \item
-If \tcode{Rep} is a scalar,
+If \tcode{Rep} is a scalar\iref{qty.set.traits},
 \tcode{\placeholder{Q}} represents the scalar quantity\irefiev{102-02-19}
 of a position vector.
+% FIXME What if `Rep` is a complex number?
 \item
-If \tcode{Rep} is a vector,
+If \tcode{Rep} is a vector\iref{qty.set.traits},
 \tcode{\placeholder{Q}} represents a position vector.
 % FIXME What if `Rep` is a tensor?
 \end{itemize}
 
-\rSec1[qty.compat]{Compatibility}
-
-\rSec1[qty.one]{Dimension one}
-
 \rSec1[qty.systems]{Systems}
 
 \rSec1[qty.chrono]{\tcode{std::chrono} compatibility}
+
+\begin{codeblock}
+namespace mp_units {
+
+template<typename Period>
+consteval auto @\exposidnc{time-unit-from-chrono-period}@()
+{
+  using namespace si;
+
+  if constexpr (is_same_v<Period, std::chrono::nanoseconds::period>)
+    return nano<second>;
+  else if constexpr (is_same_v<Period, std::chrono::microseconds::period>)
+    return micro<second>;
+  else if constexpr (is_same_v<Period, std::chrono::milliseconds::period>)
+    return milli<second>;
+  else if constexpr (is_same_v<Period, std::chrono::seconds::period>)
+    return second;
+  else if constexpr (is_same_v<Period, std::chrono::minutes::period>)
+    return minute;
+  else if constexpr (is_same_v<Period, std::chrono::hours::period>)
+    return hour;
+  else if constexpr (is_same_v<Period, std::chrono::days::period>)
+    return day;
+  else if constexpr (is_same_v<Period, std::chrono::weeks::period>)
+    return mag<7> * day;
+  else
+    return mag_ratio<Period::num, Period::den> * second;
```

Comment:
We can probably do it with prose. Also, it should be part of the SI paper, not P3045.

Reply:
Very well.
It's already done,
and _is_ part of mp-units' reference documentation.